Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM liquibase/liquibase:4.4.0
+FROM liquibase/liquibase:4.7.1
```

Comment:
What do you think about using `latest` instead of a specific tag? We should only then need to update the action version below when there are breaking changes to the container.

Reply:
That would be less maintenance on our part but using an action@version should match too, with @latest or something similar.

Reply:
The action version and liquibase versions already don't match. I feel like this would be more clear to users.